test/common/stream_info/stream_info_impl_test.cc

```diff
@@ -38,7 +38,8 @@ class StreamInfoImplTest : public testing::Test {
 protected:
   void assertStreamInfoSize(StreamInfoImpl stream_info) {
     ASSERT_TRUE(sizeof(stream_info) == 840 || sizeof(stream_info) == 856 ||
-                sizeof(stream_info) == 888)
+                sizeof(stream_info) == 888 || sizeof(stream_info) == 776 ||
+                sizeof(stream_info) == 728)
```

Comment:
Out of curiosity, why does this change?

Reply:
StreamInfo has a proto member. The size of this member has changed with the new protobuf. Some old sizes can be removed later.